### ISSUE TITLE
Implement Dazed condition

### ISSUE THREAD

**Issue by JPMeehan** [MetaMorphic-Digital/draw-steel#118]
Actor sheet should gray out/otherwise visually indicate which abilities are Triggered Actions, Free Triggered Actions, or Free Maneuvers and thus cannot be used while dazed. This shouldn't be a hard restriction but it would be a visual reminder. Validate by `actor.statuses.has('dazed')`

Probably blocked by improving the actor sheet visuals.

**Comment by cdverrett94** [comment]
So I was thinking about how to do the condition blocking abilities. I want to validate the idea before doing it.

- Add a getter on the ability for `conditionRestrictions` that would return a `Set` of conditions restricting the ability. Could also just be a string, since I don't see conditions that have an overlap.
- Would add a getter for generating a tooltip that returns something like "Dazed is restricting the usage of this ability".
- The sheet display could use like a warning icon such as https://fontawesome.com/icons/triangle-exclamation?f=classic&s=solid with the tooltip on it on the ability button

**Comment by JPMeehan** [comment]
String works; there may be a broader restrictions API we want to build (lots of things seem to turn off triggered actions as a rider) but that can be a future project that builds on top of the core conditions.

**Comment by cdverrett94** [comment]
Yeah, there's also conditions that impose prone and can't stand up. For a more generalized solution, can use AEs. So have a property on the actor like `restrictions` that would have the properties `dsid` and `type`. Could be expanded to others as well if there are examples. They can be a `Set`. And then in the ability add a getter for restricted that checks the `type` and `_dsid` against those `Set`s.

I'll use a similar method as frightened to manage setting those things in the Active Effect class.
